src/modules/apiData.js

```diff
@@ -0,0 +1,19 @@
+import { setValues, getUnits } from './set_values';
+import addImg from './addImg';
+
+const getData = async (city) => {
+  const unit = getUnits();
+  const response = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&units=${unit}&appid=70c8ad9d3ce4e9eba23caf221443c534`, { mode: 'cors' });
+  const data = await response.json();
+  const values = {};
+  values.city = data.name;
```

Comment:
**[OPTIONAL]**
* [ ] Use object destructuring to avoid repetition. we can destructure `name, weather` etc, from data.